contents/blog/posthog-vs-fullstory.md

```diff
@@ -0,0 +1,576 @@
+            <td>Sync data between platforms</td>
+        </tr>
+        <tr>
+            <td><strong>Sentry</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Ingest Sentry errors for analysis</td>
+        </tr>
+        <tr>
+            <td><strong>Segment</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Ingest events from Segment</td>
+        </tr>
+        <tr>
+            <td><strong>Amazon Redshift</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Export data to Redshift</td>
+        </tr>
+        <tr>
+            <td><strong>Amazon S3</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Export data to a S3 bucket</td>
+        </tr>
+        <tr>
+            <td><strong>Azure Blob Storage</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Export data to Microsoft Azure</td>
+        </tr>
+        <tr>
+            <td><strong>Google Cloud Storage</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Export data to GCS</td>
+        </tr>
+        <tr>
+            <td><strong>Snowflake</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Export data to Snowflake database</td>
+        </tr>
+        <tr>
+            <td><strong>Google BigQuery</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Export data to BigQuery for analysis</td>
+        </tr>
+        <tr>
+            <td><strong>RudderStack</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Sync event and person data</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+## Event tracking
```

Comment:
Event tracking section doesn't feel like it adds much other than length. I think we should move Autocapture and some other table elements into the Product analytics section and remove this dedicated section.